sdk/src/azure/storage/az_storage_blobs_blob_client.c

```diff
@@ -54,63 +54,60 @@ AZ_INLINE az_span _az_get_host_from_url(az_span url)
   int32_t const url_length = az_span_size(url);
   uint8_t const* url_ptr = az_span_ptr(url);
 
-  if (url_length >= (int32_t)(sizeof("s://h") - 1))
```

Comment:
>  What this all is about, is to remove the check that is verified by a precondition below.

Can you link to where is the precondition that is verifying this? I can't find what you are referring to

Reply:
Oh right here, got it :)

https://github.com/Azure/azure-sdk-for-c/blob/736b8005b41b4ed34976b37bc79b1f3270a2278e/sdk/src/azure/storage/az_storage_blobs_blob_client.c#L124